### PR TITLE
Add IHttpConnectionFeature.ConnectionId

### PR DESCRIPTION
**Pull request by Tratcher**
#542 
@halter73

### CODE REVIEW

File: src/Microsoft.AspNetCore.Owin/OwinConstants.cs [hunk 1]
```diff
@@ -73,6 +73,7 @@ internal static class CommonKeys
             public const string RemotePort = "server.RemotePort";
             public const string LocalIpAddress = "server.LocalIpAddress";
             public const string LocalPort = "server.LocalPort";
+            public const string ConnectionId = "server.ConnectionId";
```
Comment (davidfowl) [inline src/Microsoft.AspNetCore.Owin/OwinConstants.cs]: Did you make this up?

**Comment by davidfowl** [comment]
:shipit:

File: src/Microsoft.AspNetCore.Owin/OwinConstants.cs (hunk 1 above)
Comment (Tratcher) [inline src/Microsoft.AspNetCore.Owin/OwinConstants.cs]: Yes. It's ok, I made up many of the other ones too :-)

Comment (davidfowl) [inline src/Microsoft.AspNetCore.Owin/OwinConstants.cs]: Nice

**Pull request merged**